docker.BUILD.bazel

```diff
@@ -0,0 +1,472 @@
+load("@bazel-orfs//:cc.bzl", "cc_import_binary", "cc_import_library")
+load("@bazel-orfs//:copytree.bzl", "copytree")
+load("@bazel-orfs//:openroad.bzl", "pdk")
+load("@bazel-orfs//:stringpatch.bzl", "stringpatch_binary")
+
+filegroup(
+    name = "klayout_pymod",
+    srcs = glob([
+        "lib/klayout/pymod/**/*",
+    ]),
+    visibility = ["//visibility:public"],
+)
+
+filegroup(
+    name = "mk",
+    srcs = glob([
+        "flow/util/utils.mk",
+        "flow/util/*.pl",
+        "flow/util/*.py",
+        "flow/scripts/*.py",
+        "flow/scripts/*.script",
+        "flow/scripts/*.tcl",
+    ]),
+)
+
+filegroup(
+    name = "makefile",
+    srcs = ["flow/Makefile"],
+    data = [":mk"],
+    visibility = ["//visibility:public"],
+)
+
+pdk(
+    name = "asap7",
+    srcs = glob([
+        "flow/platforms/asap7/**/*.gds",
+        "flow/platforms/asap7/**/*.lib.gz",
+        "flow/platforms/asap7/**/*.lef",
+        "flow/platforms/asap7/**/*.lib",
+        "flow/platforms/asap7/**/*.lyt",
+        "flow/platforms/asap7/**/*.mk",
+        "flow/platforms/asap7/**/*.rules",
+        "flow/platforms/asap7/**/*.sdc",
+        "flow/platforms/asap7/**/*.tcl",
+        "flow/platforms/asap7/**/*.v",
+        "flow/platforms/common/**/*.v",
+    ]),
+    visibility = ["//visibility:public"],
+)
+
+cc_import_library(
+    name = "libortools.so.9",
+    shared_library = ":lib/libortools.so.9",
+)
+
+cc_import_library(
+    name = "libtclreadline-2.3.8.so",
+    shared_library = ":lib/libtclreadline-2.3.8.so",
+)
+
+cc_import(
+    name = "libortools",
+    shared_library = ":libortools.so.9",
+)
+
+cc_import(
+    name = "libtclreadline",
+    shared_library = ":libtclreadline-2.3.8.so",
+)
+
+cc_import_binary(
+    name = "openroad_raw",
```

Comment:
This is quite a lot of boilerplate to import the binaries from the docker image, we can probably make a macro to make it less verbose.

There is also the option of using the binaries built in https://github.com/hdl/bazel_rules_hdl/, but then we are no longer guaranteed to have `openroad`, `yosys`, `Makefile` etc. from the exact same https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts commit.